.github/workflows/pr.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        k8s: [v1.17.11, v1.18.8, v1.19.1]
```

Comment:
Did we decide to keep 1.17? Has it reached end of support?

Reply:
Sorry, meant to comment on the new version of this line, not the old one, but you get the point

Reply:
Dropping support for 1.17 is not a focus of this PR. Many people are still using older version of k8s and it's not adding any feedback times to our PR workflow for having one more version to test since checks are parallel anyways. 

I believe minimal version of k8s support is a product issue and we should discuss with @yaronp68  before making a decision in a PR.